abi/abi.go

```diff
@@ -36,31 +36,46 @@ const (
 	AttestationKeyType = 2 // (ECDSA-256-with-P-256 curve)
 	// TeeTDX  for Attestation
 	TeeTDX = 0x00000081
+	// TeeTcbSvnSize is the size of TEE_TCB_SVN field in TdQuoteBody
+	TeeTcbSvnSize = 0x10
+	// MrSeamSize is the size  of MR_SEAM field in TdQuoteBody
+	MrSeamSize = 0x30
+	// TdAttributesSize is the size  of TD_ATTRIBUTES field in TdQuoteBody
+	TdAttributesSize = 0x08
+	// XfamSize is the size of XFAM field in TdQuoteBody
+	XfamSize = 0x08
+	// MrTdSize is the size of MR_TD field in TdQuoteBody
+	MrTdSize = 0x30
+	// MrConfigIDSize is the size of MR_CONFIG_ID field in TdQuoteBody
+	MrConfigIDSize = 0x30
+	// MrOwnerSize is the size of MR_OWNER field in TdQuoteBody
+	MrOwnerSize = 0x30
+	// MrOwnerConfigSize is the size of MR_OWNER_CONFIG field in TdQuoteBody
+	MrOwnerConfigSize = 0x30
+	// RtMr0Size is the size of Runtime extendable measurement register0
+	RtMr0Size = 0x30
```

Comment:
RTMRs are meant to be the same size. I think a single RtmrSize is good enough. Also I think it should be "Rtmr" instead of "RtMr"

Reply:
Done.